src/components/organisms/UserProfileModal/UserProfileModal.tsx

```diff
@@ -141,25 +139,26 @@ const SuspectedLocation: React.FC<{
   currentVenue: WithId<AnyVenue>;
 }> = ({ user, currentVenue }) => {
   // @debt This will currently load all venues in firebase into memory.. not very efficient
```

Comment:
`@debt` comment no longer relevant once this PR lands 🎉 

```suggestion
```

Reply:
removed in 78346abc